backend/fms_core/tests/test_resources.py

```diff
@@ -298,7 +298,7 @@ def test_invalid_container_rename(self):
             ("rename_invalid.csv", ValidationError),
             ("same_rename.csv", ValidationError),
             ("same_rename_2.csv", ValidationError),
-            ("double_rename.csv", ValueError),
+            ("double_rename.csv", (ValidationError, ValueError)),
```

Comment:
Does this ValidationError have to do with the double rename or this is to catch another unrelated error ? As far as i see the double rename still raise a ValueError.

Reply:
It will also raise a Validation error, that's why I have to catch both.

Reply:
Yeah but my question is should it raise a validationError ... i am not sure it should anymore than before so it might actually be testing an error with our code ... just make sure you know where this error comes from or at least ensure it's not from an undesirable behaviour.

Reply:
import throws a ValidationError if the container barcode already exists when cleaning..

Reply:
mmm ok so the test actually tests 2 errors... maybe we can split those error cases change the barcode of the same barcode twice and in another test give it an already existing barcode name ?